README.md

```diff
@@ -21,19 +21,48 @@ There are a variety of systems that use merkle-tree and hash-chain inspired data
 
 In short: JSON documents with named merkle-links that can be traversed.
 
+## Table of Contents
+
+- [Repositories](#repositories)
+  - [Specifications](#specifications)
+  - [Implementations](#implementations)
+    - [New implementations](#new-implementations)
+  - [Other Tools](#other-tools)
+  - [Meta repos](#meta-repos)
+- [Lead](#lead)
+- [Contribute](#contribute)
+- [License](#license)
+
 ## Repositories
 
-IPLD is a multifaceted, distributed effort. Here are our current repositories:
+IPLD is a multifaceted, distributed effort.
+
+### Specifications
 
+- [specs](https://github.com/ipld/specs) - The specifications for IPLD
 - [cid](https://github.com/ipld/cid) - Self-describing content-addressed identifiers for distributed systems
+
+### Implementations
+
+- [**JavaScript:** js-ipld-dag-cbor](https://github.com/ipld/js-ipld-dag-cbor) - JavaScript implementation of the IPLD spec.
+- [**Go:** ipfs/go-ipld](https://github.com/ipfs/go-ipld) - The Go implementation of IPLD.
+- [**C:** kenCode-de/c-ipld](https://github.com/kenCode-de/c-ipld) - Implementation of the IPLD spec in C.
```

Comment:
This is probably the `dag-pb` implementation only and should be updated to `c-ipld-dag-pb`, could you reachout to @kenCode-de?